vars/runPerfTestScript.groovy

```diff
@@ -0,0 +1,45 @@
+void call(Map args = [:]) {
+    String jobName = args.jobName ?: 'distribution-build-opensearch'
+    lib = library(identifier: 'jenkins@20211123', retriever: legacySCM(scm))
+    def buildManifest = lib.jenkins.BuildManifest.new(readYaml(file: args.bundleManifest))
+    String artifactRootUrl = buildManifest.getArtifactRootUrl(jobName, args.buildId)
+
+    install_dependencies()
+    install_opensearch_infra_dependencies()
+    withAWS(role: 'opensearch-test', roleAccount: "${AWS_ACCOUNT_PUBLIC}", duration: 900, roleSessionName: 'jenkins-session') {
```

Comment:
Can we use `downloadFromS3.groovy` instead?

Reply:
The `ARTIFACT_DOWNLOAD_ROLE_NAME` configuration does not have access to the bucket path where we currently have the configuration. This serves as a good starting point for the POC runs that will be executed until all the details and locations are finalized.

Reply:
What is the bucket path?

Reply:
It's stored under the Jenkins configuration. `bundles/tests/perf-test-config`